lib/index.js

```diff
@@ -58,8 +58,10 @@ api.db = null;
 // shared collections cache
 api.collections = {};
 
+const {writeConcern} = bedrock.config.mongodb;
```

Comment:
I just made this comment accidentally on a commit instead of here, but can we eliminate:

`bedrock.config.mongodb.writeConcern` and instead just use `bedrock.config.mongodb.writeOptions.writeConcern` in its place? Then if we layer that on top of the `connectOptions` when we use it, I think it will address my concern.

Reply:
This line isn't safe -- the config can be changed on a bedrock event. This top-level `writeConcern` should be removed.

Reply:
just so this has a comment. This was addressed here: https://github.com/digitalbazaar/bedrock-mongodb/commit/163a951857899ac70ee2944a892eb1b6eda2a484